### PR TITLE
Move Azure terraform example to Standard SKU for IP addresses

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
This fixes broken LoadBalancer functionality in azure CCM (it refuses to add Basic IPs into the LB)

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind regression

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
[ACTION REQUIRED] The example Terraform configs for Azure have been migrated to use the Standard SKU for IP addresses. This is a breaking change for existing setups; in which case you should continue using your current SKU
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: examples/terraform/azure/variables.tf [hunk 1]
```diff
@@ -106,11 +106,6 @@ variable "disable_kubeapi_loadbalancer" {
 
 # Provider specific settings
 
-variable "ip_sku" {
-  default     = "Basic"
-  description = "SKU to use for IP addresses"
-}
```
Comment (xmudrii) [inline examples/terraform/azure/variables.tf:112]: We should leave this variable but default it to `Standard` so users with existing setups can eventually migrate to the new Terraform configs if they want.

File: examples/terraform/azure/main.tf [hunk 2]
```diff
@@ -132,20 +118,39 @@ resource "azurerm_network_security_group" "sg" {
...
+
 resource "azurerm_public_ip" "lbip" {
   count = local.loadbalancer_count
 
   name                = "${var.cluster_name}-lbip"
   location            = var.location
   resource_group_name = azurerm_resource_group.rg.name
-  allocation_method   = "Dynamic"
-  sku                 = var.ip_sku
+  allocation_method   = "Static"
```
Comment (xmudrii) [inline examples/terraform/azure/main.tf:152]: Can we also add a variable for this?

Comment (kron4eg) [reply examples/terraform/azure/main.tf:152]: Problem is, it doesn't work with Basic anymore. So does it make sense to parametrize it?

**Pull request merged**